src/Microsoft.Health.Fhir.R5.Core/Features/Definition/unsupported-search-parameters.json

```diff
@@ -93,19 +87,6 @@
         "http://hl7.org/fhir/SearchParameter/Questionnaire-version",
         "http://hl7.org/fhir/SearchParameter/QuestionnaireResponse-questionnaire",
         "http://hl7.org/fhir/SearchParameter/RequestGroup-instantiates-canonical",
-        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-composed-of",
-        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-depends-on",
-        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-derived-from",
-        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-predecessor",
-        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-successor",
-        "http://hl7.org/fhir/SearchParameter/ResearchDefinition-version",
-        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-composed-of",
-        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-depends-on",
-        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-derived-from",
-        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-predecessor",
-        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-successor",
-        "http://hl7.org/fhir/SearchParameter/ResearchElementDefinition-version",
-        "http://hl7.org/fhir/SearchParameter/RiskEvidenceSynthesis-version",
```

Comment:
Are these now supported?

Reply:
Need to double check this when I get a chance, this changed when switching to `BaseFhirParser.Inspector`. Possibly it was able to resolve more types